logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/appVersioning/CheckIfUpdateRequiredUseCase.kt

```diff
@@ -12,6 +14,7 @@ interface CheckIfUpdateRequiredUseCase {
 
 internal class CheckIfUpdateRequiredUseCaseImpl(private val appVersionRepository: AppVersionRepository) : CheckIfUpdateRequiredUseCase {
 
-    override suspend fun invoke(currentAppVersion: Int, blackListUrl: String): Boolean =
+    override suspend fun invoke(currentAppVersion: Int, blackListUrl: String): Boolean = withContext(KaliumDispatcherImpl.default) {
```

Comment:
Should we instantiate directly the dispatcher with `KaliumDispatcherImpl`? Or should we better inject it in the constructor so that unit tests can also pass their own test dispatchers?